network/builder/src/builder.rs

```diff
@@ -125,8 +121,8 @@ impl NetworkBuilder {
 
     pub fn new_for_test(
         chain_id: ChainId,
-        seeds: &TrustedPeerSet,
-        trusted_peers: Arc<RwLock<HashMap<PeerId, HashSet<x25519::PublicKey>>>>,
+        seeds: &PeerSet,
+        trusted_peers: Arc<RwLock<HashMap<PeerId, PeerAuthContext>>>,
```

Comment:
if we used `Peer` instead of `PeerAuthContext`, this could be `Arc<RwLock<PeerSet>>` : p

Reply:
Yeah, I thought about that, and idk why I guess I didn't merge everything in connectivity manager and work out from there, this was just less changes for it without using tuples

Reply:
`PeerAuthContext` is now dumped for `Peer`